backend/src/akvo/lumen/boundary/error_tracker.clj

```diff
@@ -0,0 +1,17 @@
+(ns akvo.lumen.boundary.error-tracker
+  (:require [akvo.lumen.component.error-tracker]
+            [raven-clj.core :as raven])
+  (:import [akvo.lumen.component.error_tracker SentryErrorTracker LocalErrorTracker]))
+
+(defprotocol IErrorTracker
+  (track [this error]))
+
+(extend-protocol IErrorTracker
+
+  SentryErrorTracker
+  (track [{:keys [dsn]} error]
+    (raven/capture dsn error))
```

Comment:
There is something i don't fully _see_ ... It's supposed that `error` is a map that contains at least one key `:message`.
See the implementation of the ring middleware:
https://github.com/sethtrain/raven-clj/blob/1.4.3/src/raven_clj/ring.clj#L5-L9

In fact `raven/capture` states:
>  "Send a message to a Sentry server.
>  event-info is a map that should contain a :message key and optional
>  keys found at https://docs.getsentry.com/hosted/clientdev/#building-the-json-packet"


https://github.com/sethtrain/raven-clj/blob/1.4.3/src/raven_clj/core.clj#L46-L50

So just passing a Throwable doesn't follow that contract?

Reply:
There is an a Sentry clojure wrapper on the official Java SDK .. https://github.com/getsentry/sentry-clj
This one _does_ support passing a `{:throwable e}` as parameter